.github/workflows/tests.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.7]
```

Comment:
Do you think it is helpful to highlight in the readme file that currently Orion is developed and fully tested in python 3.7?

Reply:
I will do that in another PR when we update the readme